luaui/Widgets/gui_top_bar.lua

```diff
@@ -1,19 +1,310 @@
+local versionString = "version 0.3.02, modified 2024-09-01"
 function widget:GetInfo()
 	return {
-		name = "Top Bar",
-		desc = "Shows Resources, wind speed, commander counter, and various options.",
-		author = "Floris",
+		name = "Top Bar with Buildpower 0.3.02",
```

Comment:
Prefer storing version in a separate `version` field

Reply:
For built-in widgets storing the version at all is pretty pointless.

Reply:
KK, will delete it

Reply:
fixed version is online